metabase/metabase
# Add back button to return to the active dashboard

Epic https://github.com/metabase/metabase/issues/29648

Adds "Back to dashboard" button that's shown when the user navigates from a dashboard to the question:
- By clicking on the card title, using drill downs etc
- Dash card menu -> Edit question

The button should stay until the user leaves QB, even the question was modified etc.

How to verify:
- Open a dashboard with questions 
- Menu -> Edit question
- Make sure there is a back button
- Navigate between simple/notebook modes, the button should stay
- Clicking on the button should return you to the dashboard
- The same should happen when clicking on the dash card title, using drill downs in visualizations etc

<img width="594" alt="Screenshot 2023-05-05 at 19 31 46" src="https://user-images.githubusercontent.com/8542534/236515290-97d94d70-32eb-46f2-b9f1-930d0271d7c3.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30584)
